### PR TITLE
Optimized truncate function

### PR DESCRIPTION
**Pull request by Newman101**
Hello.

It looks like the size function count() was called as part of the test condition in line 745 of Strings.php. 

I optimized this a bit: now the count() function is being calculated beforehand, which means that it is being called only once. 

This is a minor performance improvement as the count() is not being invoked on every iteration. 

Thank you.

### CODE REVIEW

File: src/Strings.php [hunk 1]
```diff
@@ -742,7 +742,7 @@ function truncate($input, $length, $chars = '…')
         $splits = preg_split('/([\s\n\r]+)/u', $input, null, PREG_SPLIT_DELIM_CAPTURE);
 
         $splits_length = 0;
-        for ($last_split = 0; $last_split < count($splits); ++$last_split) {
+        for ($last_split = 0, $cs = count($splits); $last_split < $cs; ++$last_split) {
```
Comment (gcds) [inline src/Strings.php]: Please $define count in separate line, and if you can fix variable names to PSR-2

Comment (Newman101) [inline src/Strings.php]: Thanks for notifying, I will fix this now.

**Comment by gcds** [comment]
You forgot the variable names but does not matter I will fix it as I am releasing a new version today. Merry Christmas and Happy New Year :tada:

**Comment by Newman101** [comment]
Thank you, Merry Christmas and Happy New Year 2016!

**Pull request merged**